src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -135,6 +135,8 @@
 
 	public boolean config_claims_firespreads;						//whether fire will spread in claims
 	public boolean config_claims_firedamages;						//whether fire will damage in claims
+
+	public boolean config_claims_lecternreading;					//reading lecterns requires access trust
```

Comment:
To better match a similar config for villagers, `config_claims_lecternReadingRequiresAccessTrust` (i.e. make it similar to config node key)

Reply:
what? you don't like my super long field name? (jk jk) 
Yeah, ill change it up right quick 👍